### ISSUE TITLE
aac support

### ISSUE THREAD

**Issue by MihaD73** [bravobit/FFmpeg-Android#22]
Hello, 

It is possible to add aac support for libffmpeg.so?
I tried to add it, but even if using fdk-aac or vo-aacenc libraries, I cannot play the aac files.
 
Many thanks.

**Comment by Brianvdb** [comment]
Which ffmpeg command are you trying to execute?

**Comment by MihaD73** [comment]
Actually, I am not using a command for ffmpeg. 
I have a project that uses libffmpeg.so, and  I implemented  ffmpeg audio-only decoding (cross-platform). I compiled ffmpeg with support for mp3, mpeg, etc and works ok. I have problems with aac part. I tried to --enable-fdk-aac at compiling ffmpeg sources and used fdk-aac 0.1.5 library (latest), it compiles without any problem, but when running the apk, and choosing aac file, it does not play correctly, as it does when fdk-aac is not included at all.
So, I am thinking  that maybe I am not compiling well the ffmpeg library. Many thanks for any hint.
